prov/verbs/src/fi_verbs.c

```diff
@@ -366,5 +366,9 @@ static void fi_ibv_fini(void)
 
 VERBS_INI
 {
+	fi_param_define(&fi_ibv_prov, "iface", FI_PARAM_STRING,
+			"prefix or full name of network interface associated "
+			"with IB device (default: ib)");
+
```

Comment:
What is the iface parameter used for? Is it to select one of many verbs interfaces available on a node?